src/parser/extract-stories.test.ts

```diff
@@ -330,6 +340,131 @@ describe('extractSource', () => {
             "hasArgs": false,
             "name": "Story1",
             "source": "<div>story 1</div>",
+            "tags": [],
+            "template": false,
+          },
+        },
+      }
+    `);
+  });
+  test('Story tags', () => {
+    expect(
+      extractStories(`
+        <script>
+          import { Story, Meta } from '@storybook/addon-svelte-csf';
+        </script>
+
+        <Meta title='test'/>
+
+        <Story name="Story1" tags={["first", "second"]} >
+          <div>story 1</div>
+        </Story>
+        `)
+    ).toMatchInlineSnapshot(`
+      {
+        "allocatedIds": [
+          "default",
+          "Story",
+          "Meta",
+        ],
+        "meta": {
+          "id": undefined,
+          "title": "test",
+        },
+        "stories": {
+          "Story1": {
+            "hasArgs": false,
+            "name": "Story1",
+            "source": "<div>story 1</div>",
+            "tags": [
+              "first",
+              "second",
+            ],
+            "template": false,
+          },
+        },
+      }
+    `);
+  });
+  test('Only one story tag', () => {
+    expect(
+      extractStories(`
+        <script>
+          import { Story, Meta } from '@storybook/addon-svelte-csf';
+        </script>
+
+        <Meta title='test'/>
+
+        <Story name="Story1" tags="single" >
```

Comment:
This isn't following CSF format, right? Asking in case if we're supposed to preserve this behavior in `v5`

Reply:
That's a good question. I think our legacy support _should_ support this, but the new API shouldn't, as this is not like regular CSF, and I only added it here because it's how Meta already works.